.github/workflows/lint_python.yml

```
@@ -0,0 +1,24 @@
+name: lint_python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: sudo apt-get update
+      - run: sudo apt-get install --fix-missing ffmpeg python3-magic python3-soundfile
+      - run: pip install bandit black codespell flake8 isort mypy pytest pyupgrade safety
+      - run: bandit --recursive --skip B101,B301,B303,B311,B403 .
+      - run: black --check . || true
+      - run: codespell --ignore-words-list="tha" --skip="*/text_tests,*/misspelling.json" || true  # Until #51 lands
+      - run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+      - run: flake8 . --count --exit-zero --max-complexity=15 --max-line-length=90 --show-source --statistics
+      - run: isort --check-only --profile black . || true
+      - run: pip install torchvision -r requirements.txt
```

The line below (`pip install -e .`) should install all the requirements in `requirements.txt`, so no need to install them here as well.

```suggestion
      - run: pip install torchvision
```